setup.cfg

```diff
@@ -46,7 +46,7 @@ exclude = venv/*,var/*,codalab/lib/beam/*
 # F401 - module imported but not used
 # E501 - line too long, exceeds max-line-length
 
-ignore = E203,E731,W503,W605
+ignore = E203,E731,W503,W605,E231
```

Comment:
E231 is a conflict between black and flake8 (regarding whitespace after commas), so I'm ignoring it in flake8 so that black prevails.